lib/grape_oauth2/responses/base.rb

```diff
@@ -40,7 +40,7 @@ def headers
 
         # Raw Rack body
         def raw_body
-          @rack_response[2].body
```

Comment:
Is it a backward compatible change?

Reply:
Hmmm. I haven't tested it. But all tests are passed.